ext/liquid_c/tokenizer.c

```diff
@@ -81,7 +81,7 @@ void tokenizer_next(tokenizer_t *tokenizer, token_t *token)
           cursor++;
           token->rstrip = 1;
         }
-        if (cursor - tokenizer->cursor > 2 + token->rstrip) {
+        if (cursor - tokenizer->cursor > (intptr_t)(2 + token->rstrip)) {
```

Comment:
`cursor` and `tokenizer->cursor` are of type `const char*` so a pointer type should be used. As for "int" and "unsigned int", it may have been compiling for 32-bit, as Ruby still seems to be a fat binary on OS X:

> $ file /usr/bin/ruby
> /usr/bin/ruby: Mach-O universal binary with 2 architectures
> /usr/bin/ruby (for architecture x86_64):	Mach-O 64-bit executable x86_64
/usr/bin/ruby (for architecture i386):	Mach-O executable i386

Technically, `ptrdiff_t` should probably be used, but I don't know if that really matters.